graylog2-web-interface/src/theme/docs/ThemeProvider.md

```diff
@@ -1,5 +1,49 @@
-`ThemeProvider` is created in `src/theme/GraylogThemeProvider.jsx` and is used in `src/index.jsx` to wrap the entire application.
+[`ThemeProvider`](https://styled-components.com/docs/api#themeprovider) is created in `src/theme/GraylogThemeProvider.jsx` and is used in `src/index.jsx` to wrap the entire application.
 
-This will allow developers to access all [`theme`](https://styled-components.com/docs/api#themeprovider) props. The `theme` object contains [`color`](#section-colors) currently, but will soon adapt to the entire layout of the application.
+This will allow developers to access all `theme` props in each component. In most cases, on a new component, you can access the `theme` prop via Styled-Components like the following:
 
-For theme variants, we are using [`styled-theming`](https://github.com/styled-components/styled-theming/tree/v2.2.0) to allow for multiple themes in the future.
+```jsx static
+const StyledElement = styled.div(({ theme }) => `
+  background-color: ${theme.color.global.contentBackground};
+`);
+```
```

Comment:
Just a nitpick, but we could add `css` here and for all  `({ theme }) => ` examples, because we always add it in our code for this case.